internal/kafka/internal/config/kafka_supported_instance_types_test.go

```diff
@@ -868,6 +868,67 @@ func TestKafkaInstanceType_HasAnInstanceSizeWithLifespan(t *testing.T) {
 
 }
 
+func TestKafkaInstanceType_GetBiggestCapacityConsumedSize(t *testing.T) {
+	tests := []struct {
+		name              string
+		kafkaInstanceType KafkaInstanceType
+		want              *KafkaInstanceSize
+	}{
+		{
+			name: "The kafka instance size with the biggest capacity consumed attribute is returned",
+			kafkaInstanceType: KafkaInstanceType{
+				Id: "t1",
+				Sizes: []KafkaInstanceSize{
+					KafkaInstanceSize{Id: "s1", CapacityConsumed: 2},
+					KafkaInstanceSize{Id: "s2", CapacityConsumed: 7},
+					KafkaInstanceSize{Id: "s3", CapacityConsumed: 5},
+				},
```

Comment:
`KafkaInstanceSize` inside array initialization is redundant
```suggestion
				Sizes: []KafkaInstanceSize{
					{Id: "s1", CapacityConsumed: 2},
					{Id: "s2", CapacityConsumed: 7},
					{Id: "s3", CapacityConsumed: 5},
				},
```

Reply:
I am aware of that but it helps me identify the types even if they are in the slice above